### PR TITLE
Interest fix

### PR DESCRIPTION
**Pull request by FelicitasBeier**
Merged glo and regional realization into one realization (select_apr20) that allows the flexible choice of either regionally different interest rates (depending on the development state) or a global interest rate. 
Moreover, bugfix in regional realization.

### CODE REVIEW

File: modules/09_drivers/aug17/preloop.gms [hunk 1]
```diff
@@ -40,7 +40,7 @@ im_pop(t_all,i) = sum(i_to_iso(i,iso),im_pop_iso(t_all,iso));
                                  im_pop_iso(t_all,iso)
                              );
 
- im_gdp_pc_ppp(t_all,i)$(
```
Comment (flohump) [inline modules/09_drivers/aug17/preloop.gms:43]: We need this for forestry soon. Please keep it

File: config/scenario_config.csv [hunk 2]
```diff
@@ -1,7 +1,11 @@
 ;cc;nocc;SSP1;SSP2;SSP3;SSP4;SSP5;SDP;BASE;NPI;NDC;coupling;emulator;input;eat_lancet_diet;SUSTAg1;SUSTAg2;SUSTAg3;SUSTAg4;SUSTAg5
 c09_pop_scenario;;;SSP1;SSP2;SSP3;SSP4;SSP5;SSP1;;;;;;;;SSP1;SSP2;SSP3;SSP4;SSP5
 c09_gdp_scenario;;;SSP1;SSP2;SSP3;SSP4;SSP5;SSP1;;;;;;;;SSP1;SSP2;SSP3;SSP4;SSP5
-c12_interest_rate;;;low;medium;high;mixed;low;low;;;;;;;;low;medium;high;mixed;low
+c12_interest_rate;;;;;;;;;;;;;;;;;;;;
+s12_hist_interest_lic;;;0.07;0.07;0.07;0.07;0.07;0.07;;;;;;;;0.07;0.07;0.07;0.07;0.07
```
Comment (flohump) [inline config/scenario_config.csv:5]: This deviates from our current defaut. s12_hist_interest_lic and s12_hist_interest_hic should be the same as s12_interest_lic and s12_interest_hic by default, i.e. s12_hist_interest_lic=0.4 and s12_hist_interest_hic=0.1

Comment (FelicitasBeier) [reply config/scenario_config.csv:5]: I did follow the low/medium/high implementation from before here. There, the interest rate from 1995-2020 was 7% and then afterwards decreased/increased to 4%/7%/10% respectively. 
I can change this, but I thought I rather stick to the current scenarios.
The default in the default.cfg is how you described above.

File: modules/09_drivers/aug17/preloop.gms (hunk 1 above)
Comment (FelicitasBeier) [reply modules/09_drivers/aug17/preloop.gms:43]: codeCheck() does not pass when I leave it as im_gdp_pc_ppp. Can I still commit it then?

Comment (tscheypidi) [reply modules/09_drivers/aug17/preloop.gms:43]: codeCheck is required to pass for all updates to the develop branch. Besides that, As the module provides gdp_pc_ppp_iso, might it be possible (and potentially more consistent to the other modules) to use this instead?

File: config/scenario_config.csv (hunk 2 above)
Comment (flohump) [reply config/scenario_config.csv:5]: Yes, that was default setting for the global realization. But the default realization was regional. Therefore, the default behaviour should reflect the previous regional realization.

File: modules/09_drivers/aug17/preloop.gms (hunk 1 above)
Comment (flohump) [reply modules/09_drivers/aug17/preloop.gms:43]: I agree with Jan. We can use im_gdp_pc_ppp_iso.

File: config/scenario_config.csv (hunk 2 above)
Comment (FelicitasBeier) [reply config/scenario_config.csv:5]: Ok, so we want the historic interest rates to be gdp_dependent as in the default case. 

What about the future interest rates? I assumed that there should still be different interest rate scenarios for the different SSPs, which is why I included it in the scenario_config.csv. Should this still be a global interest rate or should it be the same as in the default.cfg (fully gdp_dependent)? In this case I could delete the switches from the scenario_config altogether, right?

Comment (FelicitasBeier) [reply config/scenario_config.csv]: Done: There is now a description of the old global SSP-specific interest rate scenarios in the default.cfg explaining how to set it if wanted.
The default is now gdp_dependent for all.

**Pull request merged**